src/main/java/core/basesyntax/service/impl/ShopService.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import java.util.Map;
+
+public interface ShopService {
+    void process(List<FruitTransaction> fruitTransactionList);
+
+    Map<String, Integer> getStorage();
```

Comment:
make your storage a global object so that ShopService isn't tightly coupled with it and isn't associated with it